grass7/raster/r.fuzzy.logic/r.fuzzy.logic.html

```diff
@@ -47,40 +47,46 @@ <h2>OPTIONS</h2>
 <h2>OUTPUTS</h2>
 <dl>
 <dt><b>output</b></dt>
-<dd>Map containing result of two-values operations. Multivalued operations will
-be available in the future. Map is always of type FCELLS and contains values
+<dd>Map containing result of two-values operations.
+Map is always of type FCELLS and contains values
 from 0 (no membership) to 1 (full membership). Values between 0 and 1 indicate
 partial membership</dd>
 </dl>
 
 <h2>SEE ALSO</h2>
 
 <em>
-<a href="r.fuzzy.html">r.fuzzy</a>,
+<a href="r.fuzzy.set.html">r.fuzzy.set</a> addon,
+<a href="r.fuzzy.system.html">r.fuzzy.system</a> addon,
 <a href="r.mapcalc.html">r.mapcalc</a>
 </em>
 
 
 <h2>REFERENCES</h2>
 
-Zadeh, L.A. (1965). Fuzzy sets. Information and Control 8 (3): 338–353.
-doi:10.1016/S0019-9958(65)90241-X. ISSN 0019-9958.<p>
+<ul>
+<li>Jasiewicz, J. (2011). A new GRASS GIS fuzzy inference system for massive data analysis.
+Computers & Geosciences (37) 1525-1531. DOI <a href="http://10.1016/j.cageo.2010.09.008">http://10.1016/j.cageo.2010.09.008</a>
+
+<li>Zadeh, L.A. (1965). "Fuzzy sets". Information and Control 8 (3): 338-353.
+doi:10.1016/S0019-9958(65)90241-X. ISSN 0019-9958.
 
-Nov&aacute;k, Vil&eacute;m (1989). Fuzzy Sets and Their Applications. Bristol: Adam Hilger.
-ISBN 0-85274-583-4.<p>
+<li>Novák, Vilém (1989). Fuzzy Sets and Their Applications. Bristol: Adam Hilger.
```

Comment:
AFAIK this will fail: the previous HTML codes are needed.

Reply:
Can you check and confirm? On my computer an HTML file containing only "Novák, Vilém" renders properly on both links (CLI) and in Firefox. I think HTML escape codes are not needed anymore on modern systems, although I did used to need them a decade ago.

Reply:
I remember issues with g.html2man but maybe that's dated... does g.extension work ok with your updated code?

Reply:
`g.extension` works on my computer. I would be OK with committing with UTF-8 symbology in the HTML file and trust modern browsers and tools to work with it, but will revert this change if anything comes up in the future.